LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 magdaszczygiel
```

Comment:
If I were you I would put my full name here, but its your call; if you prefer a username it is fine with me.